extensions/azurecore/src/azureResource/utils.ts

```diff
@@ -403,19 +404,31 @@ export async function makeHttpRequest(account: AzureAccount, subscription: azure
 		requestUrl = `${account.properties.providerSettings.settings.armResource.endpoint}${path}`;
 	}
 
-	let response: AxiosResponse | undefined;
+	let response;
 	switch (requestType) {
 		case HttpRequestMethod.GET:
-			response = await axios.get(requestUrl, config);
+			response = await httpClient.sendGetRequestAsync<any>(requestUrl, {
+				headers: {
+					'Content-Type': 'application/json',
+					'Authorization': `Bearer ${securityToken.token}`,
+					...requestHeaders
+				}
+			})
 			break;
 		case HttpRequestMethod.POST:
-			response = await axios.post(requestUrl, requestBody, config);
+			response = await httpClient.sendPostRequestAsync<any>(requestUrl, networkRequestOptions);
```

Comment:
It would be great to not use any here and fill in the actual type (making this function itself generic if needed). See https://github.com/microsoft/azuredatastudio/pull/22270 for examples of where using any can be problematic (in addition to it just being harder to work with since you have no validation if the types change)